index.html

```diff
@@ -157,10 +163,12 @@ <h2>Lists</h2>
           <dd>A red fruit that is hard to peel.</dd>
         </dl>
         <h2>Blockquote</h2>
+        
```

Comment:
@akshayvinchurkar1 also, upon closer inspection, we can remove these empty newlines between elements, so the file is uniform.